apps/aecore/include/blocks.hrl

```diff
@@ -5,10 +5,21 @@
 -define(BENEFICIARY_PUB_BYTES, 32).
 -define(BLOCK_SIGNATURE_BYTES, 64).
 -define(OPTIONAL_INFO_BYTES, 4).
+-define(FLAG_BYTES, 4).
+
 
 -define(KEY_HEADER_TAG, 1).
 -define(MICRO_HEADER_TAG, 0).
 
+%% The flag field is a 32 bit (4 bytes) field.
+%% With the following meaning:
+%% BIT        MEANING
+%% 31         Block Type: 1 - key, 0 - micro (KEY_HEADER_TAG/MICRO_HEADER_TAG)
+%% 30         For key block: ContainsInfo 1 - true, 0 - false
+%%            For micro blocks: PoFFlag: 1 - pof_hash, 0 no pof_hash
+%% 29         HC Hole block
```

Comment:
Only for key-block, right?

Reply:
Yes, and only for POS. But it is probably a good idea not to reuse flags if not strictly necessary. We have 29 free flags, but having the flags mean different things depending on other flags in the same field (as flag 30) might lead to confusion.